flume/conf/flume.conf.template

```diff
@@ -29,5 +29,11 @@ CONFIG_FOLDER=/usr/cygnus/conf
 # Which is the config file
 CONFIG_FILE=/usr/cygnus/conf/cygnus.conf
```

Comment:
This should be, at least, a general `agent.conf`

I don't know if this should be `agent_example1.conf`